packages/hdwallet-native/src/crypto/EncryptedWallet.ts

```diff
@@ -92,9 +92,9 @@ export class EncryptedWallet {
   /**
    * Generate a new mnemonic and encrypt it with the email and password
    */
-  async createWallet() {
+  async createWallet(mnemonic?) {
```

Comment:
I would recommend a separate `importMnemonic` or `createWalletFromMnemonic` function.

The correct TypeScript type here would be `async createWallet(mnemonic?: string)`